src/components/canvas-tools/geometry-tool.tsx

```diff
@@ -13,8 +13,8 @@ import "./geometry-tool.sass";
 
 interface SizeMeProps {
   size?: {
-    width: number | null;
-    height: number | null;
+  width: number | null;
+  height: number | null;
```

Comment:
Did you mean to unindent these?

Reply:
Good catch -- another earlier change that was undone but not completely. I'll fix.